selfdrive/ui/qt/offroad/settings.cc

```diff
@@ -71,6 +71,12 @@ TogglesPanel::TogglesPanel(SettingsWindow *parent) : ListWidget(parent) {
       "In this mode openpilot will ignore lanelines and just drive how it thinks a human would.",
       "../assets/offroad/icon_road.png",
     },
+    {
+      "DisengageOnGas",
+      "Disengage openpilot On Gas Press",
```

Comment:
Disengage openpilot on Accelerator Press would be better imho, but idk if it'd fit on the c2 display.

Reply:
Driving a diesel here, not really confused by the gas pedal personally

Reply:
I personally don't see the term gas pedal being used anywhere in the automotive industry *formally*, it's usually called the accelerator pedal. [Example](https://toyotamanuals.com.au/document/landing_page/prius-50-series-owners-manual-nov-17-dec-18), [Example](https://www.tacomaworld.com/threads/accelerator-and-brake-pedals-depressed-simultaneously-error-light.557335/)

Reply:
Disable OpenPilot Disengagement on Acceleration

Reply:
> Driving a diesel here, not really confused by the gas pedal personally

Disengage when you put gas in the car if you saw the suggested icon lol

Reply:
Fair enough, I deleted the suggestion.  The speed limit icon is better.

Reply:
> Fair enough, I deleted the suggestion. The speed limit icon is better.

I agree it needs a better icon, accelerator pedal should do

Reply:
Would the shorter string `Disengage on Accel. Pedal` work?

Reply:
How does that look?

Reply:
Here's an icon I made for disengage on gas move-fast toggle a while ago.
![icon_car_pedal](https://user-images.githubusercontent.com/7284371/156390665-fd7d4f52-f3bf-4f58-8e18-4fb37e365136.png)